R/as_result_data.R

```diff
@@ -12,7 +12,7 @@
 #' @param learners (list of trained [Learner]s).
 #' @param resampling ([Resampling]).
 #' @param iterations (`integer()`).
-#' @param predictions (list of [Prediction]s).
+#' @param predictions (list of list of [Prediction]s).
```

Comment:
what's happening here? why do we need to change the internal data structure for a printer?

Reply:
There was a bug in line 48.  The new printer is called in the example and failed.

Reply:
We don't use the function so we didn't notice the bug.